Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1222,6 +1222,15 @@ working, take a look at `this KEP
     free mode, make sure that default Kubernetes services like ``kube-dns`` and ``kubernetes``
     have the required label value.
 
+Neighbor Discovery
+******************
+
+When kube-proxy replacement is enabled Cilium does L2 neighbor discovery of nodes in the cluster, in
+some setups Cilium may leave stale entries behind in the neighbor table causing ethernet frames
+between some nodes to drop. To prevent Cilium from performing this neighbor discovery and instead
+rely on the Linux kernel to discover hosts on the same L2 network you can pass the
```

Comment:
thanks!

Reply:
updated in https://github.com/cilium/cilium/commit/8b0776e755924acab39dc4b2c28bdbec792d7c12. PTAL

Reply:
It's nice to have this precision, but reading the whole paragraph, it sounds like the user is facing a choice between losing packets because of Cilium leaving stale entries _or_ because the kernel of a node has no neigh entry, so it's not really clear what's best.

Is there an advantage (like, a _lower_ chance to experience drops) when using the option? Or a recommendation such as “you may want this in _this_ case, but not in _that_ case”, that we could add to help the user understand?

(Or maybe it's just me being picky - No strong objection on the current content of the note, I'm just wondering if we can make it clearer.)

Reply:
I don't know. This is maybe going too much into the details. Could you please just add an “also” to make it clearer that both cases may lead to some packet drops? Something like (emphasis on my additions):

“**However, note that** relying on the Linux Kernel might **also** cause some packets to be dropped ...”

And then let's leave it at this?

Reply:
thanks @qmonnet , your recommendation is in https://github.com/cilium/cilium/commit/07d273ba20d93e9244af55bde920f0e7ef34861c